daprdocs/content/en/reference/api/actors_api.md

```diff
@@ -484,6 +484,16 @@ Parameter | Description
 `maxStackDepth` | A value in the reentrancy configuration that controls how many reentrant calls be made to the same actor.
 `entitiesConfig` | Array of entity configurations that allow per actor type settings. Any configuration defined here must have an entity that maps back into the root level entities.
 
+
+{{% alert title="Note" color="primary" %}}
+Actor timeouts and intervals use [Go's ParseDuration](https://pkg.go.dev/time#ParseDuration) format. You can use various string formats to represent durations. For example:
```

Comment:
To keep this consistent with the Reminders section in the same topic (higher up in the topic), I suggest the following

```suggestion
Actor settings in configuration for timeouts and intervals use [time.ParseDuration](https://pkg.go.dev/time#ParseDuration) format. You can use string formats to represent durations. For example:
```

Reply:
Also I suggest that this Request Body https://github.com/dapr/docs/blob/1018f22d622c4d676a273ccdc989864ce017a4a9/daprdocs/content/en/reference/api/actors_api.md#request-body  section in the Reminders section is also referenced from the Timers section (in this document), since they are the same format. Either cut and paste to the Timers sections or in the Timers section put  link and a comment that this format also applies here. Currently it does not mention the format for Timers